.travis.yml

```diff
@@ -21,48 +28,39 @@ jobs:
         - NODE=$(which node) make lint lint-py
 
     - name: "Compile V8"
-      language: cpp
       cache: ccache
       addons:
         apt:
           sources:
             - ubuntu-toolchain-r-test
           packages:
             - g++-6
-      install:
-        - CC='ccache gcc-6' CXX='ccache g++-6' ./configure
+      install: *ccache-setup-steps
       script:
-        - CC='ccache gcc-6' CXX='ccache g++-6' make -j2 -C out V=1 v8
+        - ./configure
+        - make -j2 -C out V=1 v8
 
     - name: "Compile Node.js"
-      language: cpp
       cache: ccache
       addons:
         apt:
           sources:
             - ubuntu-toolchain-r-test
           packages:
             - g++-6
-      install:
-        - CC='ccache gcc-6' CXX='ccache g++-6' ./configure
+      install: *ccache-setup-steps
       script:
-        - CC='ccache gcc-6' CXX='ccache g++-6' make -j2 V=1
+        - ./configure
+        - make -j2 V=1
+        - make -j1 V=1 build-addons build-js-native-api-tests build-node-api-tests
+        - python tools/test.py -j 2 -p dots --mode=release --flaky-tests=dontcare addons js-native-api node-api
+        - cp out/Release/node /home/travis/.ccache
```

Comment:
https://ci.nodejs.org/job/node-cross-compile/configure copies out the following files from the cross-compile job for use in the binary test job:
```console
BINARY_FILES="config.gypi icu_config.gypi node out/Release/node out/Release/openssl-cli"
```

I'm suprised that all the tests passed without `openssl-cli`.

Reply:
Grepping for that I only found two tests in `/sequansial` and they have:
https://github.com/nodejs/node/blob/06c803d9b929880dc21eeb5f8eb40fe17ff2e499/test/sequential/test-tls-securepair-client.js#L26-L27

Reply:
I'm guessing then that they're being skipped but our test runner treats them as passed.

I see more than a few tests in `parallel`, e.g. [`test-crypto-sign-verify.js`](https://github.com/nodejs/node/blob/06c803d9b929880dc21eeb5f8eb40fe17ff2e499/test/parallel/test-crypto-sign-verify.js#L479-L480).